charts/studio/values.yaml

```diff
@@ -1231,15 +1231,26 @@ vector-aggregator:
     enabled: true
     storageClass: ""
     accessModes: ["ReadWriteOnce"]
-    size: 64Gi
+    size: 128Gi
 
   # -- Vector command.
   command: [/bin/sh, -c]
 
   # -- Vector arguments.
   args:
     - |-
-      while sleep 60; do find /data/vector/logs -type f -mtime +7 -delete; done &
+      STORAGE=/data/vector
+      MAXIMUM_SIZE_PERCENT=80
+      OLDEST_MODIFICATION_DAYS=5
+
+      while sleep 60; do
+        find "$STORAGE" -type f -mtime +"$OLDEST_MODIFICATION_DAYS" -delete
+        until test $(df "$STORAGE" | awk 'NR==2{print $5}' | tr -d %) -lt "$MAXIMUM_SIZE_PERCENT"; do
+          # HACK: assumes directory structure (single level nesting) and all files prefixed with timestamp
+          rm -f "$(find "$STORAGE" -type f | sort -t/ -k5 | tail -n1)"
+        done
+      done &
```

Comment:
are we getting notified if it exits with non 0 or killed by a kernel?

Reply:
Nevermind, wrong answer: no, we aren't (can't easily) get notified about failures here. What we can do is print logs at least.

Reply:
no, I mean exists with non 0 for whatever reason (I don't want to analyze each command and see if it can be non 0 or not - I assume it will exit at some point with non 0 for some reason and we MUST handle this - make sure pod restarts, make sure we have logs (for the command itself), make sure we are getting notifies.

Reply:
Update: upon further thought, we shouldn't handle this especially; health checks should rely exclusively on Vector, and, if it fails to write logs for whatever reason, report it.

Reply:
Indeed, we shouldn't. But, in case it happens, I would prefer to have one and only one way to detect it, and that's Vector itself. 🤞🏼

Reply:
The whole cleanup script can fail, and I wouldn't worry too much about it; in the extreme case the whole loop crashes (unlikely as per the logic) and the pod remains active, the storage will fill and we'll notice.

Reply:
>the storage will fill and we'll notice. 

how?